docs/actions/motion.create_forwarded_amendment.md

```diff
@@ -0,0 +1,22 @@
+## Payload
+```
+{
+// Required
+    meeting_id: Id;
+    title: string;
+    lead_motion_id: Id;
+    origin_id: Id;
+
+// Optional
+    text: HTML;
```

Comment:
`text` is also in the normal motion payload. There it is required, since it is a required field in the motion.
amendments have either `text` or `amendment_paragraphs`, so these must be optional here.

The reason they are in the payload and not directly handled by the code is so that `create_forwarded_amendment` can re-use as much of the normal amendment code as possible.
It's an internal action anyway.